### PR TITLE
Madior Khoussa

### PR DESCRIPTION
**Pull request by pursuitMaddy**

### CODE REVIEW

File: snowman.js [hunk 1]
```diff
@@ -13,7 +87,13 @@ function getValidLetterGuess() {
       letter = input
     } else {
       console.log("Please enter a valid letter")
+      getValidLetterGuess()
```
Comment (Sparkadelic177) [inline snowman.js]: This isn't the best place to call this function, what happens if the user guess is a valid input? What is the use of the return in line 94 ?

File: snowman.js [hunk 2]
```diff
@@ -13,7 +87,13 @@ function getValidLetterGuess() {
...
     }
   }
-  return letter.toLowerCase()
+  word1 = letter;
+  return letter.toLowerCase();
 }
+startGame()
+playGame();
+getValidLetterGuess()
+guessChar();
```
Comment (Sparkadelic177) [inline snowman.js]: Think about how to  only call the `startGame()` here and and have the other function called within the `startGame`

File: snowman.js [hunk 3]
```diff
@@ -1,10 +1,84 @@
...
+  } else {
+    count = 0;
+    console.log(`You have tried ${count} your best!!`)
+  
+  }
+}
+
+
+function replaceLetter() {
+  let newWord1 = word1.replace(current_letter, "_");
```
Comment (Sparkadelic177) [inline snowman.js]: replace would not be the idle method to replace a letter since there is at least  more than one "_" in a given word

**Comment by pursuitMaddy** [comment]
Hello sparkY
Currently each time the letter is correct and the word has two instance it shows the alert twice and I tried to include an alert in there which let the player know the letter has already been used but it contradict in the instance there is two letter the same in the world. Hope you understand my logic. 

Best,

Maddy

**Comment by pursuitMaddy** [comment]
re-submitting

**Pull request closed without merging**